package/yast2-firewall.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Sat Apr 21 07:35:42 UTC 2018 - knut.anderssen@suse.com
+
+- AutoYaST: Use tigervnc firewalld service definition instead of
```

Comment:
AFAICT this bug is not specific to AutoYaST.

Also, the changelog wording suggests that there are two service definitions one of which is wrong so we are using the other one. But in [Bug 1088647](https://bugzilla.suse.com/show_bug.cgi?id=1088647) you say:
> The service is not called vnc-server anymore but tigervnc and tigervnc-https instead.

So the changelog should say something like this, right?
> Use the correct service name (tigervnc) for opening the VNC port in the firewall